### DISCUSSION TITLE
How to export the paperspace of .dxf to .pdf?

### DISCUSSION THREAD

**Discussion by Werner-mc1** [mozman/ezdxf#959] [Q&A]
I´m trying to export a .dxf drawing into PDF. My problem is that in the export, only the paperspace without it model-layer is shown.
**Please help**!

**Comment by mozman** [comment]
These examples show how to export modelspace and paperspace layouts as SVG/PDF by the `drawing` add-on:

- https://github.com/mozman/ezdxf/blob/master/examples/addons/drawing/svg_export.py
- https://github.com/mozman/ezdxf/blob/master/examples/addons/drawing/pdf_export.py

Tutorial how to use the new backends for the `drawing`  add-on in `ezdxf` v1.1:

- https://ezdxf.mozman.at/docs/tutorials/image_export.html

FAQs for the `Matplotlib` backend: #550
